src/ansys/fluent/core/session_meshing.py

```diff
@@ -19,19 +19,27 @@ class Meshing(PureMeshing):
     def __init__(
         self,
         fluent_connection: FluentConnection,
+        remote_file_handler: Optional[Any] = None,
     ):
         """Meshing session.
 
         Args:
             fluent_connection (:ref:`ref_fluent_connection`): Encapsulates a Fluent connection.
+            remote_file_handler: Supports file upload and download.
         """
-        super(Meshing, self).__init__(fluent_connection=fluent_connection)
+        super(Meshing, self).__init__(
+            fluent_connection=fluent_connection, remote_file_handler=remote_file_handler
+        )
+        self._remote_file_handler = remote_file_handler
```

Comment:
This should already be set from the base session

Reply:
Right,  We will remove this. Thank you.

Reply:
Done. Thank you.